### ISSUE TITLE
Missing firmware

### ISSUE THREAD

**Issue by R-3MY** [OpenIPC/device-mjsxj02hl#24]
Hi,

I have an issue with flashing due to : https://github.com/OpenIPC/firmware/blob/master/ATTENTION.md

I cannot find the May 1st 2023 firmware because their telegram channel delete messages after 6 months.

Regards, Rémy.

**Comment by jekkos** [comment]
Hi, did you find a build that works? I saw you posted on the other repo so maybe you went for another firmware? How did you flash that one in the end? Also using hitool?

**Comment by R-3MY** [comment]
Yep i already flashed my camera with a custom firmware from : https://kasito.ru/ and https://github.com/kasitoru/mjsxj02hl_firmware as i remember i have only used the sd card / reset button to flash the new firmware.

I was looking for another firmware when i was trying to flash the camera again and i saw it's no longer supported by the author.

**Comment by jekkos** [comment]
thanks for the info, Im struggling a bit to see how the SD card should be inserted in the device.  It does not snug in any way and seems to sit loose most of the time. Did you experience a similar issue?

"Rémy BOULAY" ***@***.***> schreef op 4 januari 2024 08:41:49 CET:

**Comment by R-3MY** [comment]
Nop sorry, it was fine for me with the SD card. It's like any other device with SD card, i don't remember the side don't try to push, it need to go in without effort and there is a little spring so you will ear a little click when it's done.

**Comment by jekkos** [comment]
I managed to figure out how it should go in, but wonder did you install this demo file from the sdcard using the original bootloader or the one you flashed with openipc?
